python/pyspark/sql/group.py

```diff
@@ -238,7 +238,7 @@ def pivot(self, pivot_col, values=None):
         ----------
         pivot_col : str
             Name of the column to pivot.
-        values :
+        values : list, optional
```

Comment:
Do we use this convention anywhere else?

Reply:
Yeah, it's from https://numpydoc.readthedocs.io/en/latest/format.html. I think it's fine.

Reply:
I would do it in batch though.